lib/canvas-motion.js

```diff
@@ -28,8 +30,14 @@ CanvasMotion.prototype = {
   }
 };
 
-function moveFishX(fish, round) {
-  fish.x += (fish.velocity * round);
+function moveFishX(fish, round, p5Amplitude) {
+  // console.log("moveFishX " + p5Amplitude);
+  fish.x += (fish.velocity * round/2 * p5Amplitude);
+  return fish;
+}
+
+function moveFishY(fish, p5Amplitude) {
+  fish.y = fish.y + p5Amplitude/150;
```

Comment:
This is a very random suggestion - but to make the user really understand what's going on with the sound and the fishies - you might want to make or grab a mix or clips of songs off of the Gump soundtrack - that way you'll A: avoid copywrite issues with 30 second clips and B: show velocity changing and the fish reacting to breaks in songs, etc